__tests__/app.test.js

```diff
@@ -37,3 +37,26 @@ describe('1: GET api/treasures', () => {
           })
       });
 })
+
+describe('2. GET /api/articles', () => {
+    test('status:200, responds with an array of articles objects', () => {
+      return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeInstanceOf(Array);
```

Comment:
This is the line Dave was talking about, we don't need to explicitly check for an array here as we are later using a forEach and checking the length so it must be an array right?

Reply:
Aren't you checking if it is an array before we used forEach? forEach could also be used for an object no?